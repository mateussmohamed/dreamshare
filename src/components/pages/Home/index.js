import React from 'react'

import { Page } from 'templates'
import {
  Hero,
  HowDreamshare,
  DiscoverPartners,
  SearchPartners,
} from 'organisms'

const Home = () => (
  <Page>
    <Hero />
    <HowDreamshare />
    <DiscoverPartners />
    <SearchPartners />
  </Page>
)

export default Home
