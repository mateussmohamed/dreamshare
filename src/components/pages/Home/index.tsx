import React from 'react'

import { Page } from '../../templates/Page'
import {
  Hero,
  HowDreamshare,
  DiscoverPartners,
  SearchPartners,
} from '../../organisms'

export const Home = () => (
  <Page>
    <Hero />
    <HowDreamshare />
    <DiscoverPartners />
    <SearchPartners />
  </Page>
)
