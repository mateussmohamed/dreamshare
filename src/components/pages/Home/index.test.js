import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import Home from '.'

describe('[<Home />]', () => {
  const wrap = (props = {}) => shallow(<Home {...props} />).dive()

  test('matches to snapshot', () => {
    const wrapper = wrap()

    expect(wrapper).toMatchSnapshot()
  })
})
