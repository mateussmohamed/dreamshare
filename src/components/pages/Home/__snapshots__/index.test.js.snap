// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<Home />] matches to snapshot 1`] = `
<Page__Wrapper>
  <Hero />
  <HowDreamshare />
  <DiscoverPartners />
  <SearchPartners />
  <Footer />
</Page__Wrapper>
`;
